tsdb/engine/tsm1/engine.go

```diff
@@ -667,6 +687,7 @@ func (e *Engine) WriteSnapshot() error {
 	defer func() {
 		if started != nil {
 			e.Cache.UpdateCompactTime(time.Now().Sub(*started))
+			e.traceLogger.Printf("Snapshot written in %v", time.Since(*started))
```

Comment:
I'd make this a regular logger, not trace. Also, should include the shard ID or path.